projects/igniteui-angular/src/lib/grids/api.service.ts

```diff
@@ -142,8 +142,9 @@ export class GridBaseAPIService<T extends GridType> implements GridServiceType {
             return;
         }
         const args = cell.createEditEventArgs(true);
-
-        this.grid.summaryService.clearSummaryCache(args);
+        if (!this.grid.crudService.row) {
```

Comment:
I feel the whole if may be excessive, may be is worth trying of any test will fail after removing this.